module/lib.js

```diff
@@ -81,6 +81,7 @@ export function avgD20roll(d20Roll) {
     }
     d20Roll._total = d20Roll._total + newTotal - oldTotal;
     if (!(minVer('0.8.0'))) d20Roll.results[0] = newTotal;
+    show3dDice(avgRoll);
```

Comment:
Actually, this line should be deleted entirely. Otherwise, Dice So Nice rolls twice when you run a skill check from a character sheet.  Also, the relevant portion of `chkRoll()` should be revised to read:
```js
    if (adv === 0 && (game.settings.get("grouproll", "averageRolls") === "c" || game.settings.get("grouproll", "averageRolls") === "a")) avgD20roll(roll);
    show3dDice(roll);
    return roll;
```

Reply:
ee9d5506c46e4953b582c252e306fe4cf72080f3